src/i18n/locales/en/translations.json

```diff
@@ -544,35 +544,35 @@
   "wallet.addToken.form.symbol": "Symbol",
   "wallet.addToken.form.decimals": "Amount of decimals",
   "wallet.addToken.form.assetId": "Asset ID",
-  "wallet.addToken.form.supply": "Supply on AssetHub",  
+  "wallet.addToken.form.supply": "Supply on Asset Hub",
   "wallet.addToken.form.hydrationSupply": "Supply on Hydration",
   "wallet.addToken.form.multilocation.placeholder": "Paste multillocation here",
   "wallet.addToken.form.multilocation.label": "Multilocation",
   "wallet.addToken.form.button.register.hydra": "Register on Hydration",
   "wallet.addToken.form.button.register.forMe": "Add to my UI",
-  "wallet.addToken.form.button.register.update": "Update asset data",
-  "wallet.addToken.form.button.register.acceptRisk": "I understand asset got exploited",
+  "wallet.addToken.form.button.register.update": "Update asset details",
+  "wallet.addToken.form.button.register.acceptRisk": "I understand the risk",
```

Comment:
This should be probably more explicit.

Reply:
like what?

Reply:
I mean probably in the context this one is ok since it's explained b4